### PR TITLE
Homework 05 - TfIdf - Konstantin Osipov

### PR DESCRIPTION
**Pull request by PustotaPetr**

### CODE REVIEW

**Comment by PustotaPetr** [comment]
При реализации алгоритма tf-idf получен следующий граф потока выполнения:
graph_tfidf

Набор данных для теста подсчитан с использованием библиотеки Python  sklearn.feature_extraction.text.TfidfVectorizer.
Код для подготовки данных используемых в тесте:
`from sklearn.feature_extraction.text import TfidfVectorizer
import pandas as pd
import numpy as np

corpus = [
     'This is the first document.',
     'This document is the second document.',
     'And this is the third one.',
     'Is this the first document?',]

tfidf = TfidfVectorizer(norm = 'l1', sublinear_tf=False, smooth_idf=False, use_idf=False)
tfidfres_tf = tfidf.fit_transform(corpus)

tfidf = TfidfVectorizer(norm = 'l2', sublinear_tf=False, smooth_idf=False, use_idf=True)
tfidfres_idf = tfidf.fit_transform(corpus)
idf_series = pd.Series(tfidf.idf_, index= tfidf.get_feature_names()) 

df_simple_tfidf = pd.DataFrame(tfidfres_tf.toarray(), columns=tfidf.get_feature_names(), 
                               index=range(1,5)).mul(idf_series, axis=1)

df_simple_tfidf = df_simple_tfidf.apply(np.round, decimals=3).unstack(level=0).reset_index()
df_simple_tfidf.columns = ['words', 'id', 'tfidf']
df_simple_tfidf = df_simple_tfidf[['id', 'words', 'tfidf']]
df_simple_tfidf = df_simple_tfidf.sort_values(['id', 'words'])
df_simple_tfidf = df_simple_tfidf[df_simple_tfidf['tfidf']>0]

for _, row in df_simple_tfidf.iterrows():
    print('{',row.id, ", ", '"',  row.words, '"', ", ", row.tfidf, '},',  sep='')`

**Comment by Al-p-i** [comment]
2/2
